client/src/pages/Feed.js

```diff
@@ -575,6 +575,14 @@ const Feed = (props) => {
       }
     }
   };
+
+  const onCancelModal = (currentStep) => {
+    dispatchAction(TOGGLE_STATE, "showCreatePostModal");
+    if (currentStep == 4) {
+      loadPosts();
```

Comment:
this works just fine, I think we can improve this by doing the `currentStep === 4` within the `CreatePost` component instead so we wont have to check it in all the other components.